pkg/workload/workload.go

```diff
@@ -292,5 +293,14 @@ func BaseSSAWorkload(w *kueue.Workload) *kueue.Workload {
 func AdmissionPatch(w *kueue.Workload) *kueue.Workload {
 	wlCopy := BaseSSAWorkload(w)
 	wlCopy.Status.Admission = w.Status.Admission.DeepCopy()
+	if admittedCondition := apimeta.FindStatusCondition(w.Status.Conditions, kueue.WorkloadAdmitted); admittedCondition != nil {
+		wlCopy.Status.Conditions = []metav1.Condition{*admittedCondition}
+	}
+
 	return wlCopy
 }
+
+// IsWorkloadAdmitted checks if workload is admitted based on conditions
+func IsWorkloadAdmitted(w *kueue.Workload) bool {
```

Comment:
This is enough, because this is the workload package.

```suggestion
func IsAdmitted(w *kueue.Workload) bool {
```